projects/client/RabbitMQ.Client/RabbitMQ.Client.csproj

```
@@ -24,6 +24,7 @@
     <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+    <LangVersion>latest</LangVersion>
```

@bording without this line added here and in `Unit.csproj`, VS 2017 complained that required language features were not available when trying to run individual tests from the test runner. I'm assuming it's OK to leave this in.

---

I'd go with 
```suggestion
    <LangVersion>7.3</LangVersion>
```
instead

---

Oh, and instead of setting it in each individual project, I'd suggest putting it in https://github.com/rabbitmq/rabbitmq-dotnet-client/blob/master/Directory.Build.props so it's inherited everywher.

Previously, when I suggested it to leave it out, we were discussing VS 2019 only. Is there a reason you're still using VS 2017?

---

No particular reason other than inertia 😄 I have 2019 installed and will do the migration as well as address `LangVersion`. Maybe I'll be able to just delete those lines. Stay tuned, PR on the way.